cmd/iso8583/describe.go

```diff
@@ -6,7 +6,7 @@ import (
 	"os"
 
 	"github.com/moov-io/iso8583"
-	"github.com/moov-io/iso8583/cmd/iso8583/describe"
+	msg "github.com/moov-io/iso8583/cmd/iso8583/describe"
```

Comment:
Lets rename`func describe` into `func describeMessage` and keep package as `describe`.

It's not reading well `msg.Message`, while `describe.Message` is.